### DISCUSSION TITLE
stopwatch

### DISCUSSION THREAD

**Discussion by 330305020** [SOCI/soci#1063] [Q&A] [answered]
Is there function in SOCI to output how many seconds are taken to execute sql statements? Thanks

**Comment by zann1x** [comment]
There is none I can think of, I'm sorry. You would have to roll your own implementation around SOCI for that.

**Comment by vadz** [comment, marked as answer]
There is no such function in SOCI and no reason for it to have it when it's literally 3 lines of code using C++ [chrono library](https://en.cppreference.com/w/cpp/chrono).
